src/Esfa.Vacancy.Application/Commands/CreateApprenticeship/Validators/ApplicationMethodValidator.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Esfa.Vacancy.Domain.Validation;
+using FluentValidation;
+
+namespace Esfa.Vacancy.Application.Commands.CreateApprenticeship.Validators
+{
+    public partial class CreateApprenticeshipRequestValidator
+    {
+        private void ConfigureApplicationMethodValidator()
+        {
+            RuleFor(request => request.ApplicationMethod)
+                .Cascade(CascadeMode.StopOnFirstFailure)
+                .NotEmpty()
+                .WithErrorCode(ErrorCodes.CreateApprenticeship.ApplicationMethod)
+                .Must(request => request == ApplicationMethod.Online)
+                .WithErrorCode(ErrorCodes.CreateApprenticeship.ApplicationMethod)
+                .WithMessage("Invalid Application Method");
+
+            When(request => request.ApplicationMethod == ApplicationMethod.Online, () =>
+            {
+                RuleFor(request => request.SupplementaryQuestion1)
+                    .MaximumLength(4000)
```

Comment:
Can you stick 4000 into a const as it is repeated and expected to be same for both the fields.

Reply:
Fixed